index.html

```diff
@@ -0,0 +1,314 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Coding Minds International summit</title>
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+
+</head>
+
+<body>
+
+    <main class="container">
+
+        <header class="header">
+
+            <ul class="top_info">
+                <li><a href="https://facebook.com" class="fa fa-facebook"></a></li>
+                <li><a href="https://twitter.com" class="fa fa-twitter"></a></li>
+                <li> <a class="fa" href=""> English </a> </li>
+                <li> <a class="fa" href=""> MyPage </a> </li>
+                <li> <a class="fa" href=""> Logout </a> </li>
+            </ul>        
+
+
+            <nav class="navbar">
+                <a href="index.html" alt="Coding Minds International summit Home page">
+                    <div class="logo">
+                        <img class="logo_img" src="images/logo.png" alt="conference logo">
+                        <div class="logo_title">
+                            <div class="logo_title1">CM</div>
+                            <div class="logo_title2">Summit 2015</div>
+                        </div>
+                    </div>
+                </a>
+                <div class="menu">
+                    <!-- <span clas="close">&times;</span> -->
+                    <ul class="navlist">
+                        <li> <a href="about.html" class="nav_link">About</a></li>
+                        <li> <a href="" class="nav_link">Program</a></li>
+                        <li> <a href="" class="nav_link">Join</a></li>
+                        <li> <a href="" class="nav_link">Sponsor</a></li>
+                        <li> <a href="" class="nav_link">News</a></li>
+                        <li> <a href="" class="nav_link"><button class="home_button">CM Campain</button></a></li>
+                    </ul>
+                </div>  
+            </nav>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please remove white `space` on top of your `header` part and also please `add` correct design to your main `navigation` elements like a [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/219429215-6db8d3ce-0ae6-4f44-8013-dcdbd240e21c.png)